plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -69,6 +73,14 @@ var sampleConfig = `
 
   ## Optionally provide the hex key for the IMPI connection.
   # hex_key = ""
+
+  ## If ipmitool should use a cache
+  ## for me ipmitool runs about 2 to 10 times faster enabled on HP G10 servers (when using ubuntu20.04)
+  ## the cache file may not work well for you if some sensensors come up late
+  # use_cache = true
+
+  ## Path to the ipmitools cache file (defaults to OS temp dir)	  ## Path to the ipmitools cache file (defaults to OS temp dir e.g. /tmp)
```

Comment:
This `## Path to the ipmitools cache file (defaults to OS temp dir e.g. /tmp)` seems to be redundant.